Firmware/Marlin_main.cpp

```diff
@@ -640,16 +640,9 @@ void crashdet_recover()
 	if (lcd_crash_detect_enabled()) tmc2130_sg_stop_on_crash = true;
 }
 
-void crashdet_cancel()
-{
-	saved_printing = false;
-	tmc2130_sg_stop_on_crash = true;
-	if (saved_printing_type == PRINTING_TYPE_SD) {
-		print_stop();
-	}else if(saved_printing_type == PRINTING_TYPE_USB){
-		SERIAL_ECHOLNRPGM(MSG_OCTOPRINT_CANCEL); //for Octoprint: works the same as clicking "Abort" button in Octoprint GUI
-		cmdqueue_reset();
-	}
+/// Crash detection cancels the print
+void crashdet_cancel() {
+    print_stop();
```

Comment:
The only difference I see here is that before we would send MSG_OCTOPRINT_CANCEL, but now we first call UnconditionalStop(), which clears card.sdprinting, so the condition in print_stop at ultralcd.cpp:5582 is always false...

this seems like a pre-existing bug?

Reply:
I just looked at it now and you're right!

![image](https://github.com/prusa3d/Prusa-Firmware/assets/8218499/3db13374-5742-45f2-91a8-b3234b6b83d1)
![image](https://github.com/prusa3d/Prusa-Firmware/assets/8218499/0f826640-9d21-45b6-8964-c35595affabd)

Reply:
I added another commit to resolve this hopefully.

Reply:
@gudnimg @wavexx is this solved?